recipe/bld.bat

```diff
@@ -10,13 +20,13 @@ for %%d in (2 3 RZ) do (
         -DCMAKE_BUILD_TYPE=RelWithDebInfo     ^
         -DCMAKE_C_COMPILER=clang-cl           ^
         -DCMAKE_CXX_COMPILER=clang-cl         ^
+        -DCMAKE_AR=llvm-ar.exe   ^
```

Comment:
```suggestion
        "-DCMAKE_AR=%BUILD_PREFIX%\Library\bin\llvm-ar.exe"   ^
```

Reply:
I don't think that will work: https://gitlab.kitware.com/cmake/cmake/-/issues/20180#note_678329

Do we have a forward-slash variant of `BUILD_PREFIX`?

Reply:
Is the problem solely an IPO issue? In that case, let's remove `/GL` from CXXFLAGS?

Reply:
> Is the problem solely an IPO issue?

Yes

Reply:
Hm, I cannot spot where IPO is enabled. Which flag is that? I am unsetting all CXXFLAGS and LDFLAGS hints

Reply:
problem potentially solved by setting `CMAKE_LINKER=lld-link`

https://gitlab.kitware.com/cmake/cmake/-/issues/17692